src/grouped_rects_to_place.rs

```diff
@@ -19,9 +30,9 @@ where
 {
     // FIXME: inbound_id_to_group_id appears to be unused. If so, remove it. Also remove the
     //  Hash and Eq constraints on RectToPlaceId if we remove this map
-    pub(crate) inbound_id_to_group_ids: HashMap<RectToPlaceId, Vec<Group<GroupId, RectToPlaceId>>>,
+    pub(crate) inbound_id_to_group_ids: BTreeMap<RectToPlaceId, Vec<Group<GroupId, RectToPlaceId>>>,
```

Comment:
```
// Top of file
#[cfg(test)]
use std::collections::{btree_map::Entry, BTreeMap as KeyValMap};

// Here
pub(crate) inbound_id_to_group_ids: KeyValMap<RectToPlaceId, Vec<Group<GroupId, RectToPlaceId>>>,
```

Then if `not(std)` `KeyValMap` can be an `alloc::collections::BTreeMap`

Reply:
Just noticed the part about alloc being the same as core in that std just re-exports it. Should be good to go now.